bin/subtree-split

```diff
@@ -72,6 +73,7 @@ remote wamp git@github.com:php-enqueue/wamp.git
 remote monitoring git@github.com:php-enqueue/monitoring.git
 
 split 'pkg/enqueue' enqueue
+split 'docs' php-enqueue
```

Comment:
could the repo be named docs ?

Reply:
@makasim unfortunately it's how github pages are set up - it's name has to be exactly the same as the organization.

https://help.github.com/en/articles/user-organization-and-project-pages#user-and-organization-pages-sites